fileutils/copy.go

```diff
@@ -1,14 +1,15 @@
 package fileutils
 
 import (
+	"io/fs"
 	"os"
 	"path"
 
 	"github.com/spf13/afero"
 )
 
 // Copy copies a file or folder from one place to another.
-func Copy(fs afero.Fs, src, dst string) error {
+func Copy(afs afero.Fs, src, dst string, fileMode, dirMode fs.FileMode) error {
```

Comment:
NB: Here and throughout I've had to rename these args to `afs` since the linter otherwise complains I'm conflicting with the io/fs import.